mono/metadata/domain.c

```diff
@@ -1321,6 +1323,12 @@ mono_domain_get_id (MonoDomain *domain)
 	return domain->domain_id;
 }
 
+const char *
+mono_domain_get_friendly_name (MonoDomain *domain)
```

Comment:
For this (and the functions you mention below) docs are missing because they didn't seem to be there for other similar embedding APIs in this file. Wasn't sure if that was intentional or not.